docker/php-console/entrypoint.sh

```diff
@@ -3,13 +3,4 @@ set -e
 
 setup
 
-# Wait for composer vendor
-if [ ! -f vendor/autoload.php ]; then
-    echo "Run 'composer install' command to continue."
-fi
-while [ ! -f vendor/autoload.php ]; do
-    sleep 1
-done
-echo "Composer vendor found, processing."
```

Comment:
Why removing this part?

Reply:
Why would we wait before running `tail -f`? I think this was useful before, to tell people they need to run `composer` but now it works automatically